src/components/redditClone/frontPage/index.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
```

Comment:
if you rename / move a file. Be sure to use `git mv` from the command line. Which makes sure git knows what you did. It results in a much better diff.